docs/source/api/apollo-server.md

```diff
@@ -345,10 +345,26 @@ addMockFunctionsToSchema({
 
 *  `privateVariables`: Array<String> | boolean
 
+   DEPRECATING IN VERSION XX.XX.XX for `enforcePrivateVariables`, which will support the same
+   functionalities but allow for more flexibility.
+   
    A case-sensitive list of names of variables whose values should not be sent
    to Apollo servers, or 'true' to leave out all variables. In the former
    case, the report will indicate that each private variable was redacted in
    the latter case, no variables are sent at all.
+   
+* `enforcePrivateVariables`: ((variables: Record<string, any>, operationString?: string) => Record<string, any>) | Array<String> | boolean
+    
+    A custom function for modifying variable values, a case-sensitive list of names 
+    of variables whose values should not be sent to Apollo servers, 
+    or `true` v. `false` to blacklist v. whitelist all variables, respectively. In the first
```

Comment:
I'm not familiar with the use of `v.` here.

I think we've started avoiding the terms blacklist and whitelist at Apollo. It's not always as clear as using a verb ("to mask all variables or mask no variables"), and there are unnecessary black=bad white=good implications. (Less "these are definitely offensive terms that should never be used" and more "there's usually a more specific word we can use anyway".)

Reply:
Thanks for letting me know!